src/libraries/System.Private.Uri/src/System/UriEnumTypes.cs

```diff
@@ -57,29 +57,28 @@ public enum UriFormat
     internal enum ParsingError
     {
         // looks good
-        None = 0,
+        None,
 
         // These first errors indicate that the Uri cannot be absolute, but may be relative.
-        BadFormat = 1,
-        BadScheme = 2,
-        BadAuthority = 3,
-        EmptyUriString = 4,
-        LastRelativeUriOkErrIndex = 4,
+        BadFormat,
+        BadScheme,
+        BadAuthority,
+        EmptyUriString,
+        LastRelativeUriOkErrIndex,
```

Comment:
It is equal 5 now, but it was 4. From naming it looks wrong to me. Is it as intended?